public_html/i18n/qqq.json

```diff
@@ -0,0 +1,71 @@
+{
+  "@metadata": {
+    "authors": [
+      "Niharika Kohli",
+      "MusikAnimal",
+      "Ryan Kaldari"
+    ]
+  },
+  "name": "Title of the application/tool",
```

Comment:
You may want to mark this one as optional to translate, since it's a proper noun and a translation isn't strictly needed. To mark something as optional to translate, add {{Optional}}\n to the beginning of it.